sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -2,12 +2,33 @@ CREATE TEMP FUNCTION one_index(x ANY TYPE) AS (
   CAST(IF(SAFE_CAST(x AS INT64) < 0, SAFE_CAST(x AS INT64), SAFE_CAST(x AS INT64) + 1) AS STRING)
```

Comment:
i wonder if this should be remain as an INT? are there any advantages to recasting as a string other than that's the original type?

Reply:
If it's a string, then count_picked_by_type and count_picked_by_position have the same nested types. I don't feel strongly about this either way.

Reply:
good point, let's leave it as string to reduce complexity. we can always cast to int later if needed